mesonbuild/interpreter/kwargs.py

```diff
@@ -308,3 +309,132 @@ class DoSubproject(ExtractRequired):
     version: T.List[str]
     cmake_options: T.List[str]
     options: T.Optional[CMakeSubprojectOptions]
+
+
+GNU_SYMBOL_VISIBILITY = Literal['', 'default', 'internal', 'hidden', 'protected', 'inlineshidden']
```

Comment:
Maybe offtopic, by what does `''` mean? I see the doc says that's a valid value without saying why.

Reply:
Ok, doc could be more explicit, but that's unrelated to this PR then.

Reply:
I've added a comment here in the lastest revision